catroid/src/org/catrobat/catroid/ui/dialogs/OrientationDialog.java

```diff
@@ -143,4 +122,49 @@ public void setCreateEmptyProject(boolean isChecked) {
 	public void setCreateDroneProject(boolean isChecked) {
 		createDroneProject = isChecked;
 	}
+
+	private class LoadingOnCreatingProject extends AsyncTask<String, Void, Boolean> {
```

Comment:
you are not using the return value so this could be `AsyncTask<String, Void, Void>`